src/decaylanguage/dec/dec.py

```diff
@@ -570,7 +570,7 @@ def _check_parsed_decays(self):
         """
         # Issue a helpful warning if duplicates are found
         lmn = self.list_decay_mother_names()
-        duplicates = []
+        duplicates = set()  # type: set
```

Comment:
Don't do this normally; the reason it's complaining is it wants to know what the type is in the set. `Set[<something>]` would be the proper fix, but I don't want to bother with importing typing and figuring out what is in each one of these, so I'm cheating for now by forcing it to the generic type here.

Reply:
And indeed I use a set later one, hence your change is logical.